### PR TITLE
[CPU] [ARM64] jit tanh

### PR DESCRIPTION
**Pull request by eshoguli**
### Details:
 - *[CPU] [AARCH64] jit tanh*

### Tickets:
 - *CVS-134539*

### CODE REVIEW

**Comment by dmitry-gorokhov** [comment]
@xuchen-intel Please review the PR

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp [hunk 1]
```diff
@@ -893,6 +893,86 @@ std::set<std::vector<element::Type>> jit_subtract_emitter::get_supported_precisi
...
+void jit_tanh_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (exec_prc_ != ov::element::f32) {
+        OPENVINO_THROW("unsupported precision: " + exec_prc_.to_string());
+    }
+
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+    TReg src = TReg(in_vec_idxs[0]);
+    TReg dst = TReg(out_vec_idxs[0]);
+
+    TReg aux = TReg(aux_vec_idxs[sigmoid_emitter->get_aux_vecs_count()]);
```
Comment (xuchen-intel) [inline src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp:941]: Minor: If we want to use the last element here, maybe use aux_vec_idx.back() is more straightforward.

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp [hunk 2]
```diff
@@ -893,6 +893,86 @@ std::set<std::vector<element::Type>> jit_subtract_emitter::get_supported_precisi
...
+
+    h->ld1r(aux.s, table_val2("two"));
+    h->uni_fmul(aux.s, src.s, aux.s);
+
+    std::vector<size_t> sigmoid_aux_vec_idxs;
+    sigmoid_aux_vec_idxs.assign(aux_vec_idxs.begin(), aux_vec_idxs.end());
+    std::vector<size_t> sigmoid_aux_gpr_idxs = { aux_gpr_idxs[0] };
```
Comment (xuchen-intel) [inline src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp:948]: I might miss something, but sigmoid_aux_vec_idxs and sigmoid_aux_gpr_idxs seem not used.

**Review by xuchen-intel — CHANGES_REQUESTED** [review]

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp (hunk 2 above)
Comment (tatyana-oliva) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp]: fixed, thanks

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp (hunk 1 above)
Comment (tatyana-oliva) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp]: fixed, thanks

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp (hunk 2 above)
Comment (eshoguli) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp]: cleared, thanks

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp (hunk 1 above)
Comment (eshoguli) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp]: ideally we need some connections with sigmoid emitter. but if it's not clear, then lets use your variant. was fixed, thanks

File: src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp [hunk 3]
```diff
@@ -893,6 +893,86 @@ std::set<std::vector<element::Type>> jit_subtract_emitter::get_supported_precisi
...
+    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
+        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OPENVINO_THROW("Can't create jit eltwise kernel");
+    }
+}
+
+template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+void jit_tanh_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (exec_prc_ != ov::element::f32) {
```
[the remark this answers is not in the document]
Comment (eshoguli) [reply src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp]: updated

**Review by xuchen-intel — APPROVED** [review]

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**